keyboards/3.0/fr-t-k0-azerty.xml

```diff
@@ -52,19 +52,32 @@
 	</displays>
 
 	<keys>
-		<key id="super-2" to="²" longPress="₂" />
-		<key id="amp" to="&amp;" /> <!-- 1 -->
+		<import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml" />
+		<import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+
+		<!-- switch keys -->
+		<key id="shift" switch="shift" />
+		<key id="numeric" switch="numeric" />
+		<key id="symbol" switch="symbol" />
+		<key id="base" switch="base" />
+
+
+		<key id="bksp" gap="true" /> <!-- TODO -->
+		<key id="extra" gap="true" /> <!-- TODO -->
+		<key id="enter" to="\u{000A}" /> <!-- TODO - implied key? -->
```

Comment:
The Enter key should not be generating a character -- that's an application-level decision. And certainly should not be defined in each keyboard; I can't see any need for Enter key to ever produce anything keyboard-specific (unlike spacebar).

Reply:
il'l take these keys out for now